website/docs/d/distributed_virtual_switch.html.markdown

```diff
@@ -53,8 +53,8 @@ The following arguments are supported:
   name or path.
 * `datacenter_id` - (Optional) The managed object reference ID of the
   datacenter the DVS is located in. This can be omitted if the search path used
-  in `name` is an absolute path, or if there is only one datacenter in the
-  vSphere infrastructure.
+  in `name` is an absolute path. For default datacenters, use the id attribute
+  from an empty `vsphere_datacenter` data source.
```

Comment:
This is confusing to me. What's an empty `vsphere_datacenter`

Reply:
One that is literally:

```
data "vsphere_datacenter" "dc" {}
```

This will fetch the ID of the default datacenter if it can do so (usually if on ESXi or if there just one DC in the infrastructure).

Reply:
Perfect! Makes sense